bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceAction.java

```diff
@@ -352,6 +410,24 @@ public void run() {
 		dialog.open();
 	}
 
+	private void showOverlayInEditor() {
+		Shell shellToUse = null;
+		if (overlay == null) {
+			if (fShell == null) {
+				shellToUse = fWorkbenchPart.getSite().getShell();
+			} else {
+				shellToUse = fShell;
+			}
+			overlay = new FindReplaceOverlay(shellToUse, fWorkbenchPart, fTarget);
+		}
+
+		FindReplaceOverlayFirstTimePopup.displayPopupIfNotAlreadyShown(shellToUse);
+
+		overlay.create();
```

Comment:
From the docs of Window.open, I don't even need to manually call create myself. See here:

```java
	/**
	 * Opens this window, creating it first if it has not yet been created.
	 * <p>
	 * If this window has been configured to block on open (
	 * <code>setBlockOnOpen</code>), this method waits until the window is
	 * closed by the end user, and then it returns the window's return code;
	 * otherwise, this method returns immediately. A window's return codes are
	 * window-specific, although two standard return codes are predefined:
	 * <code>OK</code> and <code>CANCEL</code>.
	 * </p>
	 *
	 * @return the return code
	 *
	 * @see #create()
	 */
```